<launch>
    <node pkg="beginner_tutorials" name="listener" type="listener">
    <param name="cont" type="string" value="logicool"/>//ps2 or logicool
    </node>
    <node pkg="joy" name="joy_node" type="joy_node"/>
    <node pkg="beginner_tutorials" name="keyboard" type="keyboard"/>
    <node pkg="beginner_tutorials" name="arduino" type="arduino"/>
</launch>